client/signup/steps/about/index.jsx

```diff
@@ -318,17 +315,13 @@ class AboutStep extends Component {
 		}
 
 		//Store
-		const isCountryAllowed =
-			includes( [ 'US', 'CA' ], this.props.countryCode ) || process.env.NODE_ENV === 'development';
 		const nextFlowName =
-			designType === DESIGN_TYPE_STORE && isCountryAllowed
-				? 'segmented-store-nux'
-				: this.props.flowName;
+			designType === DESIGN_TYPE_STORE ? 'segmented-store-nux' : this.props.flowName;
 
 		//Pressable
 		if (
 			designType === DESIGN_TYPE_STORE &&
-			( abtest( 'signupAtomicStoreVsPressable' ) === 'pressable' || ! isCountryAllowed )
```

Comment:
Are we going to remove the pressable option entirely in another PR?

Reply:
I vote to do it in another PR.